src/Microsoft.Data.SqlClient/netcore/ref/Microsoft.Data.SqlClient.cs

```diff
@@ -513,156 +351,156 @@ public sealed partial class SqlConnection : System.Data.Common.DbConnection, Sys
         public override System.Threading.Tasks.Task OpenAsync(System.Threading.CancellationToken cancellationToken) { throw null; }
         public void ResetStatistics() { }
         public System.Collections.IDictionary RetrieveStatistics() { throw null; }
-        public static void ChangePassword(string connectionString, string newPassword) { throw null; }
-        public static void ChangePassword(string connectionString, Microsoft.Data.SqlClient.SqlCredential credential, System.Security.SecureString newPassword) { throw null; }
-        public string AccessToken { get { throw null; } set { } }
-
     }
+    [System.ComponentModel.DefaultPropertyAttribute("DataSource")]
     public sealed partial class SqlConnectionStringBuilder : System.Data.Common.DbConnectionStringBuilder
     {
         public SqlConnectionStringBuilder() { }
         public SqlConnectionStringBuilder(string connectionString) { }
+        [System.ComponentModel.DisplayNameAttribute("ApplicationIntent")]
+        [System.ComponentModel.RefreshPropertiesAttribute(System.ComponentModel.RefreshProperties.All)]
         public Microsoft.Data.SqlClient.ApplicationIntent ApplicationIntent { get { throw null; } set { } }
+        [System.ComponentModel.DisplayNameAttribute("Application Name")]
+        [System.ComponentModel.RefreshPropertiesAttribute(System.ComponentModel.RefreshProperties.All)]
         public string ApplicationName { get { throw null; } set { } }
+        [System.ComponentModel.DisplayNameAttribute("Asynchronous Processing")]
+        [System.ComponentModel.RefreshPropertiesAttribute(System.ComponentModel.RefreshProperties.All)]
+        public bool AsynchronousProcessing { get { throw null; } set { } }
```

Comment:
AsynchronousProcessing does not exist in the implementation and should be removed here.

Reply:
Removed.